app/server/src/controllers/appsController.ts

```diff
@@ -14,7 +14,11 @@ export class AppsController {
             const view = `${config.appType}-app`;
             // TODO: validate config against schema for app type
             const viewOptions = {
-                appName, appTitle: config.title, courseTitle: wodinConfig.courseTitle, wodinVersion
+                appName,
+                appTitle: config.title,
+                courseTitle: wodinConfig.courseTitle,
+                wodinVersion,
+                endTime: config.endTime || 100
```

Comment:
I don't think this is working as intended. I deleted endTime from the config so `apps/day1.config.json` ended up as

```
{
  "appType": "basic",
  "title": "Day 1 - Basic Model",
  "basicProp": "day1 basic value",
  "readOnlyCode": false,
  "stateUploadIntervalMillis": 2000
}
```

then I get no plot displayed with the error `TypeError: Cannot read properties of undefined (reading 't')` coming out in the log, and with Run Options showing `NaN` as the time. Setting it there manually seems to work though. It might be worth leaving one of the example configurations missing an end time and some of the tests will pick this up, plus an explicit unit test?

Reply:
There shouldn't need to be any changes in the server code, since we're going to update endTime in the front end when we fetch config. It doesn't look like the `endTime` setting here is being used in any of the views anyway.
Agreed, `endTime` should be optional in the config,